cmd/gds/main.go

```diff
@@ -490,8 +492,17 @@ func initClient(c *cli.Context) (err error) {
 }
 
 // helper function to print JSON response and exit
-func printJSON(v interface{}) error {
-	data, err := json.MarshalIndent(v, "", "  ")
+func printJSON(m proto.Message) error {
+	opts := protojson.MarshalOptions{
+		Multiline:       true,
+		Indent:          "  ",
+		AllowPartial:    true,
+		UseProtoNames:   true,
+		UseEnumNumbers:  false,
+		EmitUnpopulated: true,
+	}
+
+	data, err := opts.Marshal(m)
```

Comment:
@rebeccabilbro made a minor change here to use `protojson` to serialize the responses rather than the `json` library - that way the enums are printed correctly. `protojson` handles data much better, so I'll probably switch to it in all of the random `printJSON` functions we have in the library!